Examples/Algorithms/Digitization/src/PlanarSteppingAlgorithm.cpp

```diff
@@ -145,27 +178,44 @@ ActsExamples::ProcessCode ActsExamples::PlanarSteppingAlgorithm::execute(
       // size_t binSerialized = binUtility.serialize({{bin0, bin1, 0}});
 
       // the covariance is currently set to 0.
-      Acts::ActsSymMatrixD<3> cov;
```

Comment:
I think the comments above should be fixed.

Reply:
Fixed.